app/controllers/verify/finance_controller.rb

```diff
@@ -48,7 +48,7 @@ def idv_finance_form
 
     def handle_success
       flash[:success] = t('idv.messages.personal_details_verified')
-      redirect_to verify_address_url
+      redirect_to verify_address_path
```

Comment:
Looks like a lazy copy from the old code, one of the methods had `url` instead of `path`